components/symbl_ai/actions/get-action-items/get-action-items.mjs

```diff
@@ -0,0 +1,30 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-get-action-items",
+  name: "Get Action Items",
+  description: "Get a list of all the action items generated from the conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/action-items)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      type: "string",
+      label: "Conversation Id",
+      description: "The Id of the Conversation",
+    },
+  },
+  async run({ $ }) {
+    try {
+      const response = await this.symblAIApp.getActionItems({
+        $,
+        conversationId: this.conversationId,
+      });
+      $.export("$summary", "Action Items successfully retrieved from the conversation");
```

Comment:
**Suggestion** (for this action and the other new actions):
Include the count of items retrieved in the [`$summary`](https://pipedream.com/docs/components/guidelines/#use-summary-to-summarize-what-happened).

```suggestion
      const { actionItems } = await this.symblAIApp.getActionItems({
        $,
        conversationId: this.conversationId,
      });
      $.export("$summary", `Successfully retrieved ${actionItems.length} Action Item${actionItems.length === 1
        ? ""
        : "s"} from the conversation`);
```

Reply:
Fixed in abb1fb0d